site/gatsby-site/src/components/checklists/ChecklistsIndex.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from 'react';
+import { Button, Spinner } from 'flowbite-react';
+import Card from 'elements/Card';
+import { Trans, useTranslation } from 'react-i18next';
+import { LocalizedLink } from 'plugins/gatsby-theme-i18n';
+import { useQuery, useMutation } from '@apollo/client';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import ExportDropdown from 'components/checklists/ExportDropdown';
+import { DeleteButton, removeTypename, statusIcon, statusColor } from 'utils/checklists';
+import { FIND_CHECKLISTS, INSERT_CHECKLIST, DELETE_CHECKLIST } from '../../graphql/checklists';
+
+export default function ChecklistsIndex() {
+  const { t } = useTranslation();
+
+  const [insertChecklist] = useMutation(INSERT_CHECKLIST);
+
+  const [deleteChecklist] = useMutation(DELETE_CHECKLIST);
+
+  const { data: checklistsData, loading: checklistsLoading } = useQuery(FIND_CHECKLISTS);
+
+  const [checklists, setChecklists] = useState([]);
+
+  useEffect(() => {
+    setChecklists(checklistsData?.checklists || []);
+  }, [checklistsData]);
+
+  if (checklistsLoading) {
+    return <Spinner />;
+  }
+  return (
+    <>
+      <div className={'titleWrapper'}>
+        <div className="w-full flex items-center">
+          <h1 className="mr-auto">
+            <Trans>Risk Checklists</Trans>
+          </h1>
+          <Button
+            onClick={() => {
+              window.location = '/apps/checklists?id=' + generateID();
+            }}
+          >
+            <Trans>New</Trans>
+          </Button>
+        </div>
+      </div>
+      <div className="flex flex-col gap-4 bg-gray-100 border-1 border-gray-200 rounded shadow-inner p-4">
+        {checklists.map((checklist) => (
+          <Card className="bg-white shadow" key={checklist.id}>
+            <div className="flex items-center gap-2 flex-wrap border-b border-gray-200 p-4">
+              <LocalizedLink className="mr-auto" to={`/apps/checklists?id=${checklist.id}`}>
+                <h2 className="mb-0">{checklist.name}</h2>
+              </LocalizedLink>
+              <Button
+                color="light"
+                onClick={async () => {
+                  const newChecklist = {
+                    ...checklist,
+                    _id: undefined,
+                    id: generateID(),
+                    name: checklist.name + t(' (Clone)'),
+                  };
+
+                  try {
+                    await insertChecklist({
+                      variables: { checklist: removeTypename(newChecklist) },
+                    });
+                    setChecklists((checklists) => {
+                      const newChecklists = [...checklists];
+
+                      newChecklists.splice(checklists.indexOf(checklist), 0, newChecklist);
+                      return newChecklists;
+                    });
+                  } catch (e) {
+                    console.log(e);
```

Comment:
Please create an issue to track improving error throwing and reporting. Ideally, also place a pointer in this code to the open issue.

Or, nix it here. :-) 
We really don't check `console.log` and I think it's a good idea.
https://github.com/search?q=repo%3Aresponsible-ai-collaborative%2Faiid+%22console.log%22+path%3A%2F%5Esite%5C%2Fgatsby-site%5C%2Fsrc%5C%2F%2F&type=code

Reply:
Replaced with toasts.